src/cmd/pay.rs

```diff
@@ -207,12 +207,15 @@ fn print_txn(
                 ]);
             }
             print_table(&table, None)?;
-
+            
+            let status_endpoint_base = "https://api.helium.io/v1/pending_transactions/";
```

Comment:
The endpoint is dynamically selected. You should be calling `api_url` defined in `mod.rs`.

Reply:
Would it be acceptable to generate the base url using api_url() when we load the wallet, and then pass this string to the print_txn() function? Or should we be generating the base url within the print_txn() function?

Reply:
I think it's probably easiest to pass it as a &str to the print_txn function. That way you maintain the logic for deriving Mainnet or Testnet in a single place (and as a bonus, you avoid another heap allocation of a String).